spec/appliance_console/cli_spec.rb

```diff
@@ -214,6 +214,46 @@
     end
   end
 
+  context "#set_server_state" do
+    it "state is status" do
+      expect(subject).to receive(:say)
+      expect(LinuxAdmin::Service).to receive(:new)
+
+      subject.parse(%w(--server status)).run
+    end
+
+    it "state is start" do
+      expect(subject).to receive(:say)
+      expect(LinuxAdmin::Service).to receive(:new)
+      expect_any_instance_of(LinuxAdmin::Service.new("evmserverd").class).to receive(:start).and_return(true)
```

Comment:
As noted [here](https://github.com/ManageIQ/manageiq-gems-pending/pull/53#issuecomment-277832314), let's try and avoid using `_any_instance_of`. Fortunately this case looks to be easy to fix as you're stubbing `new` directly before it; it doesn't matter much service type class is *actually* initialized, as you can just make whatever assertions you want on your test double:

```
context "#set_server_state" do
  let(:service) { double }

  it "state is start" do
    # ....
    expect(LinuxAdmin::Service).to receive(:new).with("evmserverd").and_return(service)
    expect(service).to receive(:start).and_return(true)
    # whatever other assertions you want
```

I know there are other specs in this file that use `any_instance_of` but we're trying to avoid using it entirely and not adding any more would make their removal a little easier 😄